Gems/MiniAudio/Code/Source/Clients/MiniAudioListenerComponentController.cpp

```diff
@@ -89,10 +154,11 @@ namespace MiniAudio
         {
             ma_engine_listener_set_position(engine, m_config.m_listenerIndex, world.GetTranslation().GetX(), world.GetTranslation().GetY(), world.GetTranslation().GetZ());
 
-            const AZ::Vector3 forward = world.TransformVector(AZ::Vector3::CreateAxisY(-1.f));
+            const AZ::Vector3 forward = world.TransformVector(AZ::Vector3::CreateAxisY(1.f));
+            const AZ::Vector3 up = world.TransformVector(AZ::Vector3::CreateAxisZ(1.f));
```

Comment:
Strange... I thought I tested left/right positioning,  was it backwards for you?

Reply:
Yes, I found that I needed to flip it.